src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -26,7 +29,7 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
      */
     val prefix = if (event.getMessage == "{}") s"[${TerminalUtil.highlight(129, "*")}] " else ""
```

Comment:
It isn't used but I also didn't want to lose this piece of knowledge (the fact that event.getMessage == `{}` for Akka actors).  Want me to comment it out or just remove it?  I guess it's really not that useful...